core/Piwik.php

```diff
@@ -128,10 +128,10 @@ public static function getPercentageSafe($dividend, $divisor, $precision = 0)
      */
     public static function getQuotientSafe($dividend, $divisor, $precision = 0)
     {
-        if ($divisor == 0) {
+        if ($divisor === 0) {
```

Comment:
I'm not sure here if `$divisor` might not be a numeric string in some cases.
```suggestion
        if ((int) $divisor === 0) {
```